lib/ZnapZend.pm

```diff
@@ -61,21 +63,48 @@ has zTime => sub { ZnapZend::Time->new() };
 
 has zLog => sub {
     my $self = shift;
+    my ($file, $syslog);
+
+    if ($self->logto){
+        #remove any wrapping quotes
+        my $logto = $self->logto;
+        if (substr($logto, -1) eq '"' && substr($logto, 0, 1) eq '"' ||
+            substr($logto, -1) eq "'" && substr($logto, 0, 1) eq "'") {
+          $logto = substr $logto, 1, -1;
+        }
+
```

Comment:
why would you want to remove quotes ? the shell will do that just fine ... such things would only be needed in dos :)